Source/WebCore/platform/graphics/filters/FEGaussianBlur.cpp

```diff
@@ -140,11 +140,27 @@ bool FEGaussianBlur::resultIsAlphaImage(const FilterImageVector& inputs) const
     return inputs[0]->isAlphaImage();
 }
 
+OptionSet<FilterRenderingMode> FEGaussianBlur::supportedFilterRenderingModes() const
+{
+    OptionSet<FilterRenderingMode> modes = FilterRenderingMode::Software;
+#if USE(GRAPHICS_CONTEXT_FILTERS)
+    if (m_stdX == m_stdY)
+        modes = modes | FilterRenderingMode::GraphicsContext;
+#endif
+    return modes;
+}
+
 std::unique_ptr<FilterEffectApplier> FEGaussianBlur::createSoftwareApplier() const
 {
     return FilterEffectApplier::create<FEGaussianBlurSoftwareApplier>(*this);
 }
 
+std::optional<GraphicsStyle> FEGaussianBlur::createGraphicsStyle(const Filter& filter) const
+{
+    auto radius = 2 * filter.resolvedSize({ m_stdX, m_stdY });
```

Comment:
Are you sure this relationship between the stdDeviation and the radius is correct?

Reply:
This is almost correct. But we can use calculateUnscaledKernelSize() which will multiply the resolved size by `3 / 4.f * sqrtf(2 * piFloat)` which is equal to 1.8799. We call this function when calculating the imageRect of the `FEGaussianBlur` and `FEDropShadow`.

I borrowed the above calculation from `FEDropShadowSoftwareApplier::apply()`.